extension/chrome/settings/index.ts

```diff
@@ -149,6 +149,16 @@ View.run(class SettingsView extends View {
         Catch.report(`Unknown target page in element: ${target.outerHTML}`);
       }
     }));
+    $('.action_open_public_key_page').click(this.setHandler(async target => {
+      const page = $(target).attr('page');
+      if (page) {
+        const ki = await KeyStore.getFirst(this.acctEmail!);
```

Comment:
@tomholub Just used `!` for testing purposes and it seems to work fine and the test passes. So is there a case when `acctEmail` is missing?

Reply:
`acctEmail` is missing when the account is not configured yet. Actually, it would be good to split this into two pages - the part of settings that are visible before an account is set up, and the part where acctEmail is always required. But that will be for another day, so it's ok to `!` bang it.

But this is not necessary:
```
 const page = $(target).attr('page');
      if (page) {
```
You can remove the `page` and other attributes from the html tag. You are now rendering it with code directly, so you don't need that. Hard-code the path into this click handler, and remove the path from the tag. That will simplify it.